it/uk/gov/hmrc/transitmovementsrouter/it/base/WiremockSuite.scala

```diff
@@ -66,6 +70,28 @@ trait WiremockSuiteWithGuice extends WiremockSuite {
       )
       .overrides(bindings: _*)
 
-  protected def bindings: Seq[GuiceableModule] = Seq.empty
+  protected lazy val injector: Injector = fakeApplication.injector
```

Comment:
we are not doing anything in it test. is this change really required?

Reply:
It is because of the need to bind the Clock in Module class and need to be applied where GuiceApplicationBuilder was bing used as it errors `- should add CustomProcessHost, X-Correlation-Id and X-Conversation-Id and Date headers to messages for GB *** FAILED ***
[info]   CreationException was thrown during property evaluation.
[info]     Message: Unable to create injector, see the following errors:
[info]   
[info]   1) No implementation for java.time.Clock was bound.
[info]     while locating java.time.Clock`. appeared all over